### ISSUE TITLE
LUA error in the Dawnbreaker dungeon when the boss casts her ability

### ISSUE THREAD

**Issue by Mrhgb** [DeadlyBossMods/DeadlyBossMods#1236]
**Describe the bug**
When the first boss in Dawnbreaker casts her ability an LUA error pops up.

**Do you have an error log of what happened?**
I couldn't get the code as it was mid fight, but I took a screenshot so hopefully it helps. 

**To Reproduce**
Go to the Dawnbreaker dungeon, do the first boss and you will get the error 100%.
-->

**Screenshots**
https://i.imgur.com/dVEQ6kP.png

**Did you try having DeadlyBossMods as the only enabled addon and everything else (especially something like ElvUI) disabled?**

No

**Which version of DeadlyBossMods are you using?**

11.0.8

**Was it working in a previous version? If yes, which was the last good one?**

No

**Additional context**

**Comment by MysticalOS** [comment]
update your dungeon mods.
it's likely you'll have to do so using wago or github or wow interface because curseforge is broken at the moment (which is likely why you have none of this weeks updates, DBM cores should also be on 11.0.10)

**Comment by Mrhgb** [comment]
> update your dungeon mods. it's likely you'll have to do so using wago or github or wow interface because curseforge is broken at the moment (which is likely why you have none of this weeks updates, DBM cores should also be on 11.0.10)

OK thank you, I didn't know Curseforge is bugged.

**Comment by MysticalOS** [comment]
Yeah, we're in communication with them but right now even though we have files on site, they are not showing to users or in curseforge app like they should be. you can actually see the files if you use legacy version of site, but they don't appear on new site

for example compare this
https://legacy.curseforge.com/wow/addons/deadly-boss-mods/files/all
to this
https://www.curseforge.com/wow/addons/deadly-boss-mods/files/all?page=1&pageSize=20
